### PR TITLE
feat(home-page): Implement first version of home page

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

- Work Type: Feature 

#### What I did

- Set up the home page with styling idea
-
-

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

**Review by davidholyko — CHANGES_REQUESTED** [review]
Revert package-lock deletion

File: src/scss/HomePage.scss [hunk 1]
```diff
@@ -1,4 +1,287 @@
+$base03: #002b36;
```
Comment (davidholyko) [inline src/scss/HomePage.scss:1]: we probably want some kind of documentation for what these colors look like. Can we add that file into our documentation folder

File: src/scss/HomePage.scss [hunk 2]
```diff
@@ -1,4 +1,287 @@
...
+.triple-header {
+  width: 100vw;
+  height: 30vh;
+  display: flex;
+  margin-left: 50px;
+  align-items: center;
+}
+
+.triple-content {
+  width: 100vw;
```
Comment (davidholyko) [inline src/scss/HomePage.scss:145]: change width to 100%. 100vw looks different on windows.

File: src/scss/HomePage.scss [hunk 3]
```diff
@@ -1,4 +1,287 @@
...
+$magenta: #d33682;
+$violet: #6c71c4;
+$blue: #268bd2;
+$cyan: #2aa198;
+$green: #859900;
+
 .home-page {
   display: flex;
   flex-direction: column;
+  overflow: scroll;
```
Comment (davidholyko) [inline src/scss/HomePage.scss:21]: change to overflow-y: overlay;

https://stackoverflow.com/questions/18548465/prevent-scroll-bar-from-adding-up-to-the-width-of-page-on-chrome

File: src/components/pages/HomePage.js [hunk 4]
```diff
@@ -1,11 +1,105 @@
...
+      <nav className="navbar">
+        <h1 className="title">ResGenie</h1>
+        <Link className="nav-link" to="editor-page">
+          Get Started!
+        </Link>
+        <Link className="nav-link" to="editor-page">
+          FAQ
+        </Link>
+      </nav>
+      <div className="hero-section">
```
Comment (davidholyko) [inline src/components/pages/HomePage.js:18]: some of these divs are semantically <section></section> elements.

File: src/components/pages/HomePage.js [hunk 5]
```diff
@@ -1,11 +1,105 @@
...
+        </div>
+        <div className="hero-two">
+          <img className="hero-image" src={FolesCoverImg} />
+        </div>
+      </div>
+      <div className="triple-column-section">
+        <div className="triple-header">
+          <h1>Create reusmes that get results! In 3 easy steps!</h1>
+        </div>
+        <div className="triple-content">
```
Comment (davidholyko) [inline src/components/pages/HomePage.js:36]: these content triplets are semantically article elements

File: src/components/pages/HomePage.js [hunk 6]
```diff
@@ -1,11 +1,105 @@
...
+      </div>
+      <div className="youtube-how-to-section">
+        <div className="video-demo">This is video demo</div>
+      </div>
+      <div className="footer-section">
+        <div className="footer-header">
+          <h3>Get going!</h3>
+          <button className="btn-one">Create your Resume!</button>
+        </div>
+        <div className="footer-content">
```
Comment (davidholyko) [inline src/components/pages/HomePage.js:61]: footer element

File: src/scss/HomePage.scss (hunk 2 above)
Comment (PatrickDohn) [reply src/scss/HomePage.scss:145]: All fixed

File: src/scss/HomePage.scss (hunk 3 above)
Comment (PatrickDohn) [reply src/scss/HomePage.scss:21]: Taken care of!

Comment (PatrickDohn) [reply src/scss/HomePage.scss]: All set

File: src/components/pages/HomePage.js (hunk 4 above)
Comment (PatrickDohn) [reply src/components/pages/HomePage.js]: Taken care of

File: src/components/pages/HomePage.js (hunk 5 above)
Comment (PatrickDohn) [reply src/components/pages/HomePage.js:36]: Taken care of

File: src/components/pages/HomePage.js (hunk 6 above)
Comment (PatrickDohn) [reply src/components/pages/HomePage.js:63]: Taken care of

File: src/scss/HomePage.scss (hunk 2 above)
Comment (PatrickDohn) [reply src/scss/HomePage.scss]: Taken care of

**Review by davidholyko — APPROVED** [review]
LGTM

**Pull request merged**